pandasai/config.py

```diff
@@ -32,6 +33,12 @@ def load_config(override_config: Config = None):
     if override_config:
         config.update(override_config)
 
+    if "llm" in config:
+        try:
+            config["llm"].is_pandasai_llm()
+        except AttributeError:
+            config["llm"] = LangchainLLM(config["llm"])
+
```

Comment:
[Follow-up of what we are discussing in #509]
This should work fine, except that a user who wants to actually use pandasai's types and instantiate a `Config`  object to initialize their `SmartDataFrame` needs to manually do the conversion to `LangchainLLM`, as we are not doing that at the level of the `Config`. If we are good with this, I believe this is a good solution! :)

Reply:
I'd say at the moment there is not real reason to encourage users to instantiate their own `Config` (we are indeed trying to move configs to the json so that less and less config is required over time).

I'n not even in love with the idea of using pydantic validation to do transformations, that still seems a little bit of an anti-pattern to me. I'd say let's keep it this way for now (meaning that an advanced user that wants to instantiate a `Config`, will have no problem at figuring out how to instantiate a `LangChainLLM` wrapper).

If we figure out this is not the case, we might try to handle that edge case better. What do you think?